lib/vsc/utils/exceptions.py

```diff
@@ -35,13 +35,13 @@
 from vsc.utils import fancylogger
 
 
-def get_callers_logger():
+def get_callers_logger(use_inspect=False):
```

Comment:
`get_callers_logger(use_inspect=True)`, how else?

it's not set in the call done from `LoggedException`, which is fine, but having control is nice?